packages/http/src/isomorphic.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference lib="dom" />
```

Comment:
Probably a lack of knowledge on my part, but why is this file named "isomorphic"? It seems like it's a polyfill, or a list of namespaces that map to either web-browser-maintained packages, or node packages.

Is "isomorphic" the standard JS term for this? I don't know that there's a better name necessarily, I'm just curious.